backend/development.ini.sample

```diff
@@ -35,6 +35,7 @@ basic_setup.website_base_url = http://localhost:7999
 basic_setup.listen = localhost:6543
 ## storage path/connection
 basic_setup.sqlalchemy_url = sqlite:///%(here)s/tracim.sqlite
+basic_setup.depot_storage_type = local
```

Comment:
Why use `depo_....` name here and not `basic_setup.uploaded_files_storage_type`?

Reply:
I ask because S3 is not depot (discussed today), so I really not understand depo = s3

Reply:
I agree that it does not look coherent

Reply:
renamed.